### ISSUE TITLE
CSS conflicts with numerous aspects

### ISSUE THREAD

**Issue by Aetherinox** [MrGVSV/obsidian-github-embeds#5]
I like the plugin, however, I've noticed that the plugin seems to conflict with how certain things appear.

One header I have should look like the following:
![IPInKoo](https://github.com/MrGVSV/obsidian-github-embeds/assets/118329232/3555571f-17a6-4758-8f7a-3577f3f1b035)

However, as soon as I enable this plugin
![wHSRKAQ](https://github.com/MrGVSV/obsidian-github-embeds/assets/118329232/748b81bf-dd2d-4b6c-b5dd-7208426caddf)

And that's just a regular old link to an external website.

I also noticed that the following CSS property seems to make some things way bigger than they should be:

```css
._embed-container_tpqtd_1~._embed-container_tpqtd_1{
    margin-top:var(--size-4-4)
}
```

I haven't seen exactly what it is associated to, but my embedded gits seem to be ok so far, I had to override it with
```css
._embed-container_tpqtd_1
{
    margin: auto !important;
}
```

**Comment by MrGVSV** [comment]
Yeah seems to be related to #3. I'll try to take a look soon!

**Comment by Aetherinox** [comment]
Thanks.

Also, not a  huge thing, but if some day you got the chance to integrate gists into this addon, it would make a perfect single solution.

I am currently using https://github.com/linjunpop/obsidian-gist and it's a great plugin, but having everything wrapped into one would be the icing on the cake. Also sort of irritating that the gist plugin has no support for dark themes. So I have this dark Obsidian theme and then when I scroll to a gist, I get blasted with white.

Not a deal breaker, but just a thought.

**Comment by MrGVSV** [comment]
@Aetherinox could you provide some more details such as:
- Current theme
- List of active plugins
- Minimum markdown needed to reproduce the issue

Thanks!

**Comment by Aetherinox** [comment]
|  | |
| --- | --- |
| Theme | Default |
| Plugins | Dataview, Github Embeds |

Everything is pretty much stock except for having the dataview plugin enabled. Maybe it's happening because it's sort of unique how I have it setup as I'm using frontmatter values.

Install dataview and create a new markdown page with the following:

```markdown
---
title: "Obsidian: Gist"
name: "Obsidian: Gist"
description: Render the [GitHub Gist](https://gist.github.com) in [Obsidian](https://obsidian.md).
modified: 2023-11-01T21:13:44-07:00
---

# `! this.name`
`! this.description`

---
```

You should have the following page:
![eZpSuXf](https://github.com/MrGVSV/obsidian-github-embeds/assets/118329232/0dae5f7f-922a-47b5-95be-a68801443951)

The other change that needs made is inside the Dataview settings, change the `Inline Query Prefix` to `!` and at the top of the dataview settings, and **enable** the `Inline Queries`

![hIVVYUh](https://github.com/MrGVSV/obsidian-github-embeds/assets/118329232/b3d3cf4e-9815-427c-bd8a-9a3b4234e175)

---
With Github Embeds disabled, I get:
![FmYo2RS](https://github.com/MrGVSV/obsidian-github-embeds/assets/118329232/ccb32f33-a9b2-44c1-9bc1-14a222b3ae29)

If Github Embeds is enabled:
![D4cLt5l](https://github.com/MrGVSV/obsidian-github-embeds/assets/118329232/0462417f-52a8-4665-bcc6-1f237c5ae078)

**Comment by MrGVSV** [comment]
What version of Obsidian and Dataview are you on? I'm trying to reproduce:

| Source | Edit | Reader |
| ------ | ---- | ------ |
| <img width="781" alt="source_view" src="https://github.com/MrGVSV/obsidian-github-embeds/assets/49806985/faa36594-e514-4c7a-b21f-20ee2c9589ec"> | <img width="781" alt="edit_view" src="https://github.com/MrGVSV/obsidian-github-embeds/assets/49806985/a16f45c9-d9fd-4108-9acb-a37d5c3df072"> | <img width="781" alt="reader_view" src="https://github.com/MrGVSV/obsidian-github-embeds/assets/49806985/5a195ae9-bb52-4a17-bfb9-fc2cfd11ccff"> |

And it looks like the images above with GE both enabled and disabled.

**Comment by Aetherinox** [comment]
Obsidian: 1.4.16
Dataview: 0.5.63

![8ixn0pS](https://github.com/MrGVSV/obsidian-github-embeds/assets/118329232/a162a19a-d4d1-419c-b94e-89311244d275)

![Bz998ls](https://github.com/MrGVSV/obsidian-github-embeds/assets/118329232/487f058a-4a27-4d15-9873-31cc0b0aca96)

**Comment by MrGVSV** [comment]
Oh you know what? I've been testing with #6 already merged. Checking out a commit before then does reveal your same issue.

So this is actually fixed!

**Comment by MrGVSV** [comment]
Fixed by #6

**Comment by MrGVSV** [comment]
@Aetherinox I just released[ v1.0.3](https://github.com/MrGVSV/obsidian-github-embeds/releases/tag/1.0.3). Let me know if your issue still persists after the update once it's available for download!

**Comment by Aetherinox** [comment]
Sorry for the delay. I've got a lot of Git notifications.
I'll throw up a test page and report back. Thanks!
